protocol/contracts/beanstalk/silo/ConvertFacet.sol

```diff
@@ -176,14 +523,14 @@ contract ConvertFacet is ReentrancyGuard {
                 stems,
                 amounts,
                 a.active.tokens,
-                bdvsRemoved
+                a.bdvsRemoved
             );
 
             emit LibSilo.TransferBatch(
                 LibTractor._user(), 
                 LibTractor._user(),
```

Comment:
they are only memory loads and they are hot. i do not agree that this is a worthwhile change, as it will cascade throughout the codebase degrading readability in exchange for dust

Reply:
Disagree on the readability - `account` vs `LibTractor._user()` have the same readability imo (I'd argue that `account` is a little easier to read). For newer developers who aren't fully aware of how `LibTractor_user()` works, it may be confusing for someone to see the function called multiple times when the return value doesn't change. Defer to @pizzaman1337 here